scripts/incremental/utils.py

```diff
@@ -0,0 +1,246 @@
+import os
+import shutil
+from pathlib import Path
+import subprocess
+from pydriller import Git
+import re
+import pandas
+import json
+import numpy as np
+import brokenaxes
+import matplotlib as mpl
```

Comment:
Might be good to have a `requirements.txt` file for all the non-standard dependencies.